global_config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "sdkVersion": "1.8", // The version of the Answers SDK to use
+  "sdkVersion": "develop", // The version of the Answers SDK to use
```

Comment:
is this one intentional too?

Reply:
yes, we'll switch it to 1.9 when that's released (or we can switch it to the release branch if we cut those)

Reply:
updated to point to release/v1.9